pmm/kubernetes-cluster-staging.groovy

```diff
@@ -146,10 +146,7 @@ pipeline {
                                 set -o errexit
                                 set -o xtrace
                                 sudo yum -y install curl
-                                sudo curl -Lo /usr/local/sbin/minikube https://github.com/kubernetes/minikube/releases/download/v1.16.0/minikube-linux-amd64
-                                sudo chmod +x /usr/local/sbin/minikube
-                                sudo ln -s /usr/local/sbin/minikube /usr/sbin/minikube
-                                alias kubectl='minikube kubectl --'
+                                /srv/pmm-qa/pmm-tests/install_k8s_tools.sh --minikube --kubectl --sudo
```

Comment:
We have Jenkins agent where kubectl already was pre-installed:
https://github.com/percona/pmm-infra/blob/main/packer/ansible/agent.yml
Can we migrate to this agent?

I don't really like the approach where we download tools every time :(

Reply:
maybe, from other point - how often we update the image and the tool in the image is also downloaded (not from distr) so you would need to bump version quite often.